### PR TITLE
Updating some episode titles and tour collections

### PR DESCRIPTION
**Pull request by mbrandonw**

### CODE REVIEW

File: Sources/Models/Episode.swift [hunk 1]
```diff
@@ -147,6 +147,7 @@ public struct Episode: Equatable, Identifiable {
     }
 
     public struct Section: Equatable {
+      public var alternateSlug: String?
```
Comment (mbrandonw) [inline Sources/Models/Episode.swift:150]: collection sections now support alternate slugs so we can rename their titles.

File: Sources/Transcripts/Collections/Tours.swift [hunk 2]
```diff
@@ -36,9 +21,10 @@ extension Episode.Collection {
           .init(episode: .ep249_tourOfTCA),
         ],
         related: [],
-        title: "Composable Architecture 1.0",
+        title: "The Composable Architecture",
```
Comment (mbrandonw) [inline Sources/Transcripts/Collections/Tours.swift:24]: Dropping the 1.0 from the tour section.

File: Sources/Transcripts/Collections/Tours.swift [hunk 3]
```diff
@@ -64,6 +50,21 @@ extension Episode.Collection {
...
+        blurb: #"""
+        We open sourced the [Composable Architecture](http://github.com/pointfreeco/swift-composable-architecture) after many, _many_ months of developing the concepts from first principles in Point-Free episodes. In this tour we build a small application from scratch, focus on breaking it down to into small understandable units, and write a full test suite to exercise every subtle aspect of the application.
+        """#,
+        coreLessons: [
+          .init(episode: .ep100_ATourOfTheComposableArchitecture_pt1),
+          .init(episode: .ep101_ATourOfTheComposableArchitecture_pt2),
+          .init(episode: .ep102_ATourOfTheComposableArchitecture_pt3),
+          .init(episode: .ep103_ATourOfTheComposableArchitecture_pt4),
+        ],
+        isHidden: true,
```
Comment (mbrandonw) [inline Sources/Transcripts/Collections/Tours.swift:63]: this just moved below. it's the 0.x tour and it's hidden but kept around to preserve links.

File: Sources/Transcripts/Episodes/0243-TourOfTCA.swift [hunk 4]
```diff
@@ -26,7 +27,7 @@ extension Episode {
     ],
     sequence: 243,
     subtitle: "The Basics",
-    title: "Tour of the Composable Architecture 1.0",
+    title: "Tour of the Composable Architecture",
```
Comment (mbrandonw) [inline Sources/Transcripts/Episodes/0243-TourOfTCA.swift:30]: dropping 1.0 from all episodes

File: Sources/Transcripts/Episodes/0247-TourOfTCA.swift [hunk 5]
```diff
@@ -16,8 +17,8 @@ extension Episode {
       .scrumdinger,
     ],
     sequence: 247,
-    subtitle: "Correctness",
-    title: "Tour of the Composable Architecture 1.0",
+    subtitle: "Domain Modeling",
```
Comment (mbrandonw) [inline Sources/Transcripts/Episodes/0247-TourOfTCA.swift:20]: new subtitle

**Review by stephencelis — APPROVED** [review]
Looks good!

**Pull request merged**